src/oops/SOLID/openClosePrinciple/before/client/CalculateTaxesClient.java

```diff
@@ -20,16 +22,16 @@ public static void main(String[] args) {
         // Calculate taxes
         Locale locale = new Locale("en", "US");
         NumberFormat currencyFormatter = NumberFormat.getCurrencyInstance(locale);
-        TaxCalculator taxCalculator = new TaxCalculator();
 
         double totalTaxes = 0;
         for (Employee employee: employees){
 
+            TaxPayable taxPayable = TaxPayableFactory.createPayableObj(employee);
```

Comment:
For someone new, the name TaxPayable could be confused to contain the tax amount. Therefore, is slightly misleading. Also, we generally need not not use Obj in the method names.

Reply:
Noted. Could have done a better job at naming.